catalog/app/containers/Auth/saga.js

```diff
@@ -415,6 +415,20 @@ function* handleCheck(
   }
 }
 
+/**
+ * Handle GET_TOKENS action.
+ *
+ * @param {Action} action
+ */
+function* handleGetTokens({ meta: { resolve, reject } }) {
+  try {
+    const tokens = yield call(getTokens)
+    yield call(resolve, tokens)
+  } catch (e) {
+    if (reject) yield call(reject, e)
```

Comment:
Probably, we need to add `console.error` or `Sentry.captureExeption` to handle miss of `reject`